private/rules/pom_file.bzl

```diff
@@ -1,33 +1,41 @@
 load(":has_maven_deps.bzl", "MavenInfo", "has_maven_deps")
 
-_PLAIN_DEP = """        <dependency>
-            <groupId>{0}</groupId>
-            <artifactId>{1}</artifactId>
-            <version>{2}</version>
-        </dependency>"""
+def _format_dep(unpacked):
+    return "".join([
+ "        <dependency>\n",
```

Comment:
Indentation goes wrong here. Have you run `buildifier -r -lint fix  .` on this PR? If not, could you please do so?

Reply:
running on whole repo creates big diff, so applying it only to this file for now